Shared/Samples/Show device location with NMEA data sources/SimulatedNMEASentenceFeed.swift

```diff
@@ -13,10 +13,9 @@
 // limitations under the License.
 
 import Foundation
-import ArcGIS
 
-/// A data source simulating NMEA data.
-class SimulatedNMEAData {
+/// A data source simulating a hardware that emits NMEA data.
+class SimulatedNMEASentenceFeed {
```

Comment:
This type may use a better name, please advice.

Reply:
I would say something like `FileNMEASentenceReader`